Makefile

```
@@ -0,0 +1,34 @@
+all: test
+
+venv: .built/venv
+
+.built/venv: .built/
+	( \
+		python3 -m venv venv ;\
+		source venv/bin/activate ;\
+		pip install --upgrade pip ;\
+		pip install --upgrade -r requirements.txt ;\
+		touch $@ ;\
+	)
+
+test: venv
+	( \
+		source venv/bin/activate ;\
```

Shouldn't this be `.` so it is POSIX compatible? I'm unsure at the moment but commands usually run with `/bin/sh` and it doesn't have the `source` alias.

```suggestion
		. venv/bin/activate ;\
```